src/graphql/cluster.rs

```diff
@@ -294,8 +294,9 @@ struct ColumnTimeSeries<'a> {
 
 #[Object]
 impl ColumnTimeSeries<'_> {
-    async fn column_index(&self) -> usize {
-        self.inner.column_index
+    // The column index of the time series in string within the representable range of `usize`
```

Comment:
`//` It would be better to change to `///`.
And it is recommended to modify the document to end with a "."